src/VisualStudio/Core/Def/Implementation/SymbolSearch/RoslynSymbolSource.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editor;
+using Microsoft.CodeAnalysis.Editor.FindUsages;
+using Microsoft.CodeAnalysis.Editor.GoToBase;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.VisualStudio.Core.Imaging;
+using Microsoft.VisualStudio.Language.Intellisense.SymbolSearch;
+
+namespace Microsoft.VisualStudio.LanguageServices.SymbolSearch
+{
+    internal class RoslynSymbolSource : ISymbolSourceFromLocation
+    {
+        internal SymbolSourceProvider ServiceProvider { get; private set; }
+
+        public RoslynSymbolSource(SymbolSourceProvider symbolSourceProvider)
+        {
+            ServiceProvider = symbolSourceProvider;
+        }
+
+        public string UniqueId => "Roslyn symbol source";
```

Comment:
>"Roslyn symbol source"; [](start = 34, length = 23)

move to const?
you consume it twice here and in other files as well

Reply:
it's a property on the interface